src/components v2/Atoms/Forms/InputDate/InputDate.vue

```diff
@@ -8,7 +8,9 @@
       :maxlength="2"
       :value="value.day"
       :disabled="disabled"
+      :is-number="true"
```

Comment:
`is-number` w/o the `true` should suffice no?

Reply:
yeap